tests/pdk/finfet_pdk/test_placement_grid.py

```diff
@@ -16,6 +16,7 @@ def test_scalings():
     name = f'ckt_{get_test_id()}'
     netlist = textwrap.dedent(f"""\
     .subckt dig22inv a o vccx vssx
+    * @: Generator(pattern='CS')
```

Comment:
What is this for?

Reply:
Is there any way we can make the generator query case insensitive?